Assets/Global/Scripts/Player/Player.cs

```diff
@@ -1,3 +1,4 @@
+using System.Collections.Generic;
```

Comment:
this is added, but not used?

there's also the UIElements import that is unused, but thats not added in this PR.